### PR TITLE
Security armor and hardsuits can store the blunderbuss

### PR DESCRIPTION
**Pull request by DamianX**
:cl:
 * tweak: Security armor and hardsuits can store the blunderbuss.

### CODE REVIEW

**Comment by Sooose** [comment]
*sounds of bolas flying through the air intensifies*

File: code/modules/clothing/suits/armor.dm [hunk 1]
```diff
@@ -1,5 +1,18 @@
 /obj/item/clothing/suit/armor
-	allowed = list(/obj/item/weapon/gun/energy,/obj/item/weapon/reagent_containers/spray/pepper,/obj/item/weapon/gun/projectile,/obj/item/ammo_storage,/obj/item/ammo_casing,/obj/item/weapon/melee/baton,/obj/item/weapon/handcuffs,/obj/item/weapon/gun/lawgiver,/obj/item/weapon/gun/siren,/obj/item/weapon/gun/mahoguny,/obj/item/weapon/bikehorn/baton)
```
Comment (Probe1) [inline code/modules/clothing/suits/armor.dm:2]: Well that is just awful.

**Comment by Probe1** [comment]
I hate that list.  It should have a size define and accept medium size weapons.  But that's neither here nor there.

**Comment by Blithering** [comment]
It does raise the question of who would use hard suit storage for pepper spray or a single bullet casing

**Comment by Probe1** [comment]
I _really_ hate this code.

**Pull request merged**